wiki/Beatmap_search/de.md

```diff
@@ -90,6 +91,8 @@ Metadaten von Beatmaps können für eine genauere Filterung über die folgenden
 | `length` | [Länge](/wiki/Beatmap/Play_time) in Sekunden |
 | `key`, `keys` | Anzahl der Keys (betrifft nur osu!mania und konvertierte Beatmaps) |
 | `status` | Beatmap-Status. Der Wert kann `ranked`, `approved`, `pending`, `notsubmitted`, `unknown` oder `loved` sein bzw. `r`/`a`/`p`/`n`/`u`/`l` in Kurzform. |
+| `played`, `lastplayed` | Die Zeit, seit wann zuletzt gespielt wurde. Das gültige Format ist `#y#M#d#h#m#s`, entsprechend für Jahre, Monate, Tage, Stunden, Minuten und Sekunden. Zum Beispiel bedeutet `2d5s` "2 Tage und 5 Sekunden". |
+| `divisor` | Der Nenner des [Taktteilers](/wiki/Client/Beatmap_editor/Beat_snap_divisor) |
```

Comment:
I noticed you've used `Taktteiler` a few times now when referring to the beat snap divisor. The article should be updated to at the very least include it as an alternative term (`Der Beat Snap Divisor (auch Taktteiler)`) or replace the Fremdwort entirely.

Reply:
I didn't try myself on the article so far you mention there (spoiler: the translation is heavily outdated and a mess).

The usage comes from the official stable client translation (which I wasn't involved in) and also dates back to older wiki articles (which I wasn't involved in either. I only joined the wiki later.).

You can see this in:

- [Client/Beatmap editor](https://osu.ppy.sh/wiki/de/Client/Beatmap_editor) (shows the official stable translation)
- [Old version of Client/Beatmap editor/Menu](https://github.com/ppy/osu-wiki/blob/5a2e0522222574571880f9b0b31ba624da708f58/wiki/Client/Beatmap_editor/Menu/de.md)
- [Current version of Guides/osu!mania Mapping Guide](https://github.com/ppy/osu-wiki/blob/1f2871e1c172ba74274603db2ee0ff27457c27d3/wiki/Guides/osu!mania_Mapping_Guide/de.md)

I can add this term in the article as a follow-up (without changing the rest of Beat snap divisor).

Reply:
If the main sources for this translation are stable and old wiki, it should probably be reevaluated as a translation. Not saying it's necessarily bad, but it could be